lib/ui/gallery_app_bar_widget.dart

```diff
@@ -107,6 +115,37 @@ class _GalleryAppBarWidgetState extends State<GalleryAppBarWidget> {
     );
   }
 
+  Future<dynamic> _renameAlbum(BuildContext context) async {
+    if (widget.type != GalleryAppBarType.owned_collection) {
+      return;
+    }
+    var result = await showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return ChangeCollectionNameDialog(name: appBarTitle);
+      },
+      barrierColor: Colors.black.withOpacity(0.85),
+    );
+    if (result == null) {
```

Comment:
Why would result be `null`?

Reply:
When user cancels the dialog or don't change the name, we are returning null to indicate no action is required.

Reply:
I thought I had deleted this comment.